src/mono/msbuild/android/build/AndroidApp.targets

```diff
@@ -120,7 +120,10 @@
   </Target>
 
 
-  <Target Name="_AndroidGenerateAppBundle" DependsOnTargets="_AndroidGenerateRuntimeConfig">
+  <Target
+    Name="_AndroidGenerateAppBundle"
+    Condition="$(_SkipAndroidGenerateAppBundle) != 'true'"
```

Comment:
The WasmApp.targets makes a property `IsBrowserWasmProject` that is only true if the OutputType is not Library, by default. and then gates all the wasm-specific build logic based on that.

https://github.com/dotnet/runtime/blob/a734aa467f1bbf4d5bf30fe217e1a36eb966ce08/src/mono/wasm/build/WasmApp.targets#L100

Maybe something like that should be done for Android too?  Although I didn't look if the test runners are Exe or Library and if all the functional tests are exe's, too.

/cc @radical

Reply:
I looked into the option of using `OutputType` but from what I understand we can't use it because at least the library tests don't set the `OutputType` and afaik the default is `Library`.

Reply:
How does it work on wasm then?  

It shouldn't matter if the library tests are `OutputType=Library` - the test runner is an exe, right?  

But I'm not sure what step is responsible for taking a `FOO.Tests.dll` and the apropriate test runner and putting them together into a single app.

Reply:
I think from a managed perspective it's all going to be a library.  

@simonrozsival I would a property similar to https://github.com/dotnet/runtime/blob/031177b8878371f081f8840ff814503970a7e5b0/eng/testing/tests.wasm.targets#L5

Reply:
We explicitly set the properties for the test projects:
https://github.com/dotnet/runtime/blob/0e12347861940389153ee2233ff398d551f86711/eng/testing/tests.wasm.targets#L4-L5

Reply:
I replaced the `_SkipAndroidGenerateAppBundle` with `AndroidGenerateAppBundle`.